eq-author/src/App/page/Design/answers/BasicAnswer/index.js

```diff
@@ -83,6 +85,7 @@ StatelessBasicAnswer.propTypes = {
   showDescription: PropTypes.bool,
   autoFocus: PropTypes.bool,
   getValidationError: PropTypes.func,
+  type: PropTypes.string,
```

Comment:
Should some of these PropTypes be required?

Reply:
I've added a default prop instead.